src/components/confirm/confirm.spec.js

```diff
@@ -98,8 +100,41 @@ describe("Confirm", () => {
       });
     });
 
+    it("should not render confirm button", () => {
```

Comment:
This should be wrapped in a describe block, i.e. `describe("when isLoadingProp set to true" ...` or similar